oak_runtime/src/proxy.rs

```diff
@@ -53,6 +53,7 @@ use crate::node::Node;
 pub struct RuntimeProxy {
     pub runtime: Arc<Runtime>,
     pub node_id: NodeId,
+    pub node_name: String,
```

Comment:
I don't think this should belong here; the only reason why `NodeId` is here is because it is used as an argument to partially apply to the underlying functions from the Runtime object. For now, could you add a comment mentioning that this is only used for debugging in log statements? I think we should still try and move most / all of the relevant log statements out of RuntimeProxy and into Runtime itself eventually.

Reply:
Agree with both of your suggestions, adding a TODO for it pointing to #1737